Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -670,13 +670,21 @@ extension TabDisplayManager: GroupedTabDelegate {
 
 // MARK: - InactiveTabsDelegate
 extension TabDisplayManager: InactiveTabsDelegate {
-    func shouldCloseInactiveTab(tab: Tab) {
+    func shouldCloseInactiveTab(_ tab: Tab, index: Int) {
```

Comment:
Could you please rename to `closeInactiveTabs` because the should gives the idea of decision making and that we could have a case where the tab is not closed but the code doesn't include any path for that (I know this is not related to your changes but it will improve the func intention)